types.go

```diff
@@ -17,5 +17,7 @@ type Statistics struct {
 	CpusLimit          float64 `json:"cpus_limit"`
 	MemRssBytes        int64   `json:"mem_rss_bytes"`
 	MemLimitBytes      int64   `json:"mem_limit_bytes"`
+	DiskUsedBytes      int64   `json:"disk_used_bytes"`
```

Comment:
What do you think about adding `omitempty` like ```DiskUsedBytes      int64   `json:"disk_used_bytes,omitempty"` ```? If you haven't switched on the disk isolator this won't show up, right?